.travis.yml

```diff
@@ -39,7 +37,7 @@ before_install:
 
 before_script:
   # Installs TYPO3
-  - composer require typo3/cms=$TYPO3_VERSION
+  - composer require typo3/cms-core=$TYPO3_VERSION
```

Comment:
I would prefer using the nimut/typo3-complete similar to the Crawler extension, then we have a minimal TYPO3.
But if it works without the typo3/cms-core if fine by me.

Reply:
Why load a full TYPO3 when the unittests pass with just the core? 
When adding more dependencies you should adjust the composer.json aswell. 
The core is now modular we should respect that.

Reply:
Agree, as long as it works.